### DISCUSSION TITLE
Modifying link preview without modifying page content

### DISCUSSION THREAD

**Discussion by EmilHvitfeldt** [quarto-dev/quarto-cli#8829] [Q&A] [answered]
### Description

I would like to be able to customize the link preview text for a page from a book.

concrete example. For [feaz-book.com/numeric-logarithms](http://feaz-book.com/numeric-logarithms), I get `Feature Engineering A-Z - 2 Logarithms` because it is chapter 2, but Ideally I would like it drop the chapter number in the preview so It shows up as `Feature Engineering A-Z - Logarithms`

**Comment by mcanouil** [comment, marked as answer]
Have you tried to set `pagetitle`?

**Reply by EmilHvitfeldt** [reply]
Thank you! that does appear to do the trick!
